one_shot_scripts/add_deck_names.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+const Deck = require('../models/deck');
+const Cube = require('../models/cube');
+const User = require('../models/user');
+const mongosecrets = require('../../cubecobrasecrets/mongodb');
+const {
+	generate_short_id,
+} = require('../serverjs/cubefn.js');
+
+(async () => {
+    var i = 0;
+	mongoose.connect(mongosecrets.connectionString).then( async (db) => {
+        const cubes = await Cube.find({owner: "5cf9cdf2aefc6508c5ebcb40"});
```

Comment:
What's going on here? Is this supposed to fix all the cubes? If so it probably needs some performance improvements

Reply:
Right now it is a bit hardcoded to only fix my cubes, after release I'm going to rerun so it fixes all cubes. It is absurdly slow, will likely take about 10hrs, but it's tricky because for each entry I need to make 2 DB calls... maybe I can cache some values?

Reply:
You should be able to batch them like in `validate_colors`.

Reply:
I'll revise it to be a combination of batching and caching